docs/components/webapp.rst

```diff
@@ -43,6 +43,53 @@ Then you can start and stop the webapp, adjust files, and debug.
 FIXME(willkg): Review everything after this line.
 
 
+Setting up authentication and a superuser
+=========================================
+
+Creating a superuser
+--------------------
+
+If you want to do anything in the webapp admin, you'll need to create a
+superuser.
+
+Run this::
+
+  $ docker-compose run webapp python manage.py createsuperuser
```

Comment:
It doesn't. If the user doesn't exist, it errors out:

https://github.com/mozilla-services/socorro/blob/master/webapp-django/crashstats/authentication/management/commands/makesuperuser.py#L25

We talked about this earlier. You said we should change to something like what Tecken does--that's a good idea. I figured it's easier for now to just do a docs change.